### PR TITLE
updating the contact form

### PR DESCRIPTION
**Pull request by RuriEnciso**
We update the contact form from the mobile version. 
- It´s on the last page

### CODE REVIEW

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @RuriEnciso 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request 🔴 

You have submitted a project, that has some parts that makes it unprofessional. Kindly complete the Future features section, and delete all recommended comments on the README such as that shown in the picture below:

## Image
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: styles.css [hunk 1]
```diff
@@ -274,240 +210,347 @@ textarea::placeholder {
...
-    padding-top: 20%;
-    padding-bottom: 20%;
-    background-color: #fff;
-  }
+form {
+  margin: 0 auto;
+  padding-top: 20%;
+  padding-bottom: 20%;
+  background-color: #fff;
+}
```
Comment (Zuhaib042) [inline styles.css]: - [ ]  Kindly consider laying out the form elements using flexbox as it is a project requirement.
  -  In order to lay out the elements on the page you should use Flexbox.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @RuriEnciso  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

## Highlights
✅ Professional Readme
✅ Correct Github flow

## Required Changes ♻️
- [ ] Kindly consider making sure that the name field in the form only accepts up to 30 characters. you can apply that using `maxlength="30"` attribute inside this tag of yours `<input type="text" id="name" name="name" class="input-padron" required placeholder="Full name">`
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by souad988 — APPROVED** [review]
Hi @RuriEnciso , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6a4a5143327075565a70544d4f34765573302f67697068792e676966


## Highlights
✔Good use of github workflow
✔No linter errors
✔Overall work is good keep the hard work 💪
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**